src/StoreApi/README.md

```diff
@@ -66,7 +66,9 @@ Available resources in the Store API are listed below, with links to more detail
 |                                                              | `GET`, `POST`, `PUT`, `DELETE` | [`/wc/store/v1/cart/items/:key`](docs/cart-items.md#single-cart-item)                         |
 | [`Cart Coupons`](docs/cart-coupons.md)                       | `GET`, `POST`, `DELETE`        | [`/wc/store/v1/cart/coupons`](docs/cart-coupons.md#list-cart-coupons)                         |
 |                                                              | `GET`, `DELETE`                | [`/wc/store/v1/cart/coupon/:code`](docs/cart-coupons.md#single-cart-coupon)                   |
-| [`Checkout`](docs/checkout.md)                               | `GET`, `POST`                  | [`/wc/store/v1/checkout`](docs/checkout.md)                                                   |
+| [`Checkout`](docs/checkout.md)                               | `GET`, `POST`                  | [`/wc/store/v1/checkout`](docs/checkout.md)     
+| [`Checkout order`](docs/checkout-order.md)                   | `POST`                         | [`/wc/store/v1/checkout`](docs/checkout-order.md)                                             |
```

Comment:
```suggestion
| [`Checkout order`](docs/checkout-order.md)                   | `POST`                         | [`/wc/store/v1/checkout/:id`](docs/checkout-order.md)                                             |
```
I just noticed that the path parameter `:id` is missing in the endpoint example. We should mention that to be considtent with the examples of the other endpoints.